Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10-slim
+FROM python:3.10-slim-bookworm
```

Comment:
why this change?

Reply:
I mentioned it in a comment above, the Microsoft drivers we're installing have to target a debian version, so explicitly calling this out to avoid future conflicts